static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -131,13 +149,31 @@ function DefaultGroupEventDetailsContent({
         />
       </StyledDataSection>
       {event.userReport && (
-        <EventDataSection title={t('User Feedback')} type="user-feedback">
-          <EventUserFeedback
-            report={event.userReport}
-            orgSlug={organization.slug}
-            issueId={group.id}
-            showEventLink={false}
-          />
+        <EventDataSection
+          title={t('User Feedback')}
+          type="user-feedback"
+          actions={
+            <ErrorBoundary mini>
+              <ButtonBar gap={1}>
+                <HideUserFeedbackButton
+                  isHidden={feedbackHidden}
+                  showFeedback={showPrompt}
+                  hideFeedback={dismissPrompt}
+                />
+              </ButtonBar>
+            </ErrorBoundary>
+          }
+        >
+          {feedbackHidden ? (
+            '(Hidden)'
```

Comment:
style thing but could we instead have something like `This feedback is hidden.` in grey text and italicized? i think that would match sentry style a bit more